monitoring/prometheus/app.js

```diff
@@ -73,9 +75,13 @@ app.get('/', (req, res) => {
 });
 
 // [START monitoring_sli_metrics_prometheus_metrics_endpoint]
-app.get('/metrics', (req, res) => {
-  res.set('Content-Type', prometheus.register.contentType);
-  res.end(prometheus.register.metrics());
+app.get('/metrics', async (req, res) => {
+  try {
+    res.set('Content-Type', register.contentType);
+    res.end(await register.metrics());
```

Comment:
Tiny nit: @fhinkel is this appropriate, or would it be better to break the `await` into a separate line (to avoid confusion as to what `res.end` accepts)?

Reply:
I think that's fine.